app/helpers/application_helper.rb

```diff
@@ -410,7 +410,6 @@ def git_branch_info
   end
 
   def exportable_admin_metadata_fields
-    (DulHydra.user_editable_admin_metadata_fields +
-        DulHydra.user_editable_item_admin_metadata_fields).map { |f| Ddr::Index::Fields.get(f) }
+    Ddr::Index::Fields.adminmd - [Ddr::Index::Fields::IS_LOCKED, Ddr::Index::Fields::ACCESS_ROLE]
```

Comment:
@dchandekstark I'm concerned that this creates a mismatch between what can be exported in a metadata file and what is acceptable in an uploaded metadata file.  E.g., this appears to allow workflow_state_ssi to exported but MetadataFileUpload will not accept a metadata that contains a workflow_state_ssi column, since it is not an editable admin metadata field and is not on the list of non-editable metadata fields permitted by MetadataFileUpload.  So, if you want to keep this change, I think we'll need to make a change to MetadataFileUpload as well (or else metadata editors will have to remember to remove that column before round-tripping an exported metadata file).  If you want to keep this change but don't want to make changes to MetadataFileUpload part of this PR, would you at least make an issue for changing MetadataFileUpload and pin it to Dec release so we're sure it gets done before that release is cut?  Thanks.